schema/me/conversation/index.js

```diff
@@ -319,6 +322,18 @@ export const ConversationFields = {
       })
     },
   },
+
+  unread: {
+    type: GraphQLBoolean,
+    description: "True if there is an unread message by the user.",
+    resolve: conversation => {
+      const memoizedLastMessageId = lastMessageId(conversation)
+      const { from_last_viewed_message_id } = conversation
+      return (
+        !!from_last_viewed_message_id && !!memoizedLastMessageId && from_last_viewed_message_id < memoizedLastMessageId
```

Comment:
Once these values are present, (so there has been an initial gallery reply), that means we are tracking this conversation properly and can do a simple comparison to determine if there are any messages the user has not seen.